static/js/components/PersonalForm.js

```diff
@@ -39,6 +39,37 @@ export default class PersonalForm extends ProfileFormFields {
     ui:                     UIState,
   };
 
+  renderRomanizedFields = () => {
+    const { profile } =  this.props;
+    const english = /^[A-Za-z]*$/;
+    let firstName = _.get(profile, ["first_name"], "");
+    let lastName = _.get(profile, ["last_name"], "");
+    let rFirstName = _.get(profile, ["romanized_first_name"], "");
+    let rLastName = _.get(profile, ["romanized_last_name"], "");
+    const showRomanizedFields = (
+      !english.test(firstName) || !english.test(lastName) || rFirstName || rLastName
```

Comment:
This code looks fine to me - my comment before was regarding the regex being used, it only allows lowercase/uppercase characters a-z whereas the original issue requested characters in the CP-1252 range (which is what Pearson requires), which includes many characters with diacritical marks. For example, the last name `Müller` is a common last name in Germany and fits in CP-1252, but doesn't pass the regex above.

My previous comment detailed the Unicode ranges I think are worth validating against.